### DISCUSSION TITLE
[Docs] moco v3 No print loss

### DISCUSSION THREAD

**Discussion by weikai520** [open-mmlab/mmselfsup#535] [How-to] [answered]
### 📚 The doc issue

No loss was printed while training with moco v3 today

### Suggest a potential alternative/fix

Because of too little data set can't print, can be in mmselfsup/configs/selfsup/base/default runtime. Py modify logger = dict (type = 'LoggerHook', the interval = 5)

**Comment by Jiahao000** [comment, marked as answer]
Hi, the default log interval in `default_runtime.py` is 50, you can customize it by yourself if your dataset is too small.

**Comment by tonysy** [comment]
Thanks.
This Q & A  should be helpful for other community users. I would like to summarize the solution:

Change the logger with smaller interval like:
```python
logger = dict (type = 'LoggerHook', the interval = 5)
```
